src/editor/components/modals/revert-modal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import format from "date-fns/format";
+
+import Modal from "../../../shared/components/modal";
+import ModalTitle from "../../../shared/components/modal-title";
+import ModalContent from "../../../shared/components/modal-content";
+import ModalCall from "../../../shared/components/modal-call";
+import { TextButton } from "../../../shared/components/buttons";
+
+export default class RevertModal extends React.Component {
+  static propTypes = {
+    onCloseOrCancel: PropTypes.func.isRequired,
+    onRevert: PropTypes.func.isRequired,
+    visible: PropTypes.bool,
+    aboveOtherModals: PropTypes.bool,
```

Comment:
I don't think there is any need to define this (or pass it in)? I think it's always true.